js/modules/k6/experimental/streams/tests/testharness.js.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/resources/testharness.js b/resources/testharness.js
+index c5c375e17..a4610d927 100644
+--- a/resources/testharness.js
++++ b/resources/testharness.js
+@@ -2589,10 +2589,6 @@
+      */
+     Test.prototype.step = function(func, this_obj)
+     {
+-        if (this.phase > this.phases.STARTED) {
+-            return;
+-        }
+-
+         if (settings.debug && this.phase !== this.phases.STARTED) {
+             console.log("TEST START", this.name);
+         }
+@@ -2621,9 +2617,6 @@
+         try {
+             return func.apply(this_obj, Array.prototype.slice.call(arguments, 2));
+         } catch (e) {
+-            if (this.phase >= this.phases.HAS_RESULT) {
+-                return;
+-            }
+             var status = e instanceof OptionalFeatureUnsupportedError ? this.PRECONDITION_FAILED : this.FAIL;
+             var message = String((typeof e === "object" && e !== null) ? e.message : e);
+             var stack = e.stack ? e.stack : null;
+@@ -2631,6 +2624,8 @@
+             this.set_status(status, message, stack);
+             this.phase = this.phases.HAS_RESULT;
+             this.done();
++
++            throw `${this.name} failed - ${e}`;
```

Comment:
I am not certain this changes make sense, can you explain them?

Reply:
To be honest, I haven't fully got how it works yet, but the management of the whole tests suite, its phases and all that, messes it up and prevents tests from being executed by running a single file, as we do. Again, all because of the assumptions it has on running on a browser, how the feedback is returned, etc.

Reply:
I think I finally managed to get it working, _coming soon_!